### ISSUE TITLE
Keyboard dissapears while scrolling omnibox suggestions

### ISSUE THREAD

**Issue by Y-T-G** [kiwibrowser/src.next#286]
**Describe the bug**
When you scroll through the suggestions shown in omnibox, the keyboard will dissappear

**To Reproduce**
1. Enable bottom bar (don't know if required)
2. Set kiwi://flags/#omnibox-ui-max-autocomplete-matches to 'Enabled 12 matches' (to increase the max number of suggestions shown')
3. Type a keyword and try scrolling

**Expected behavior**
Keyboard should stay on-screen

**Screenshots**
![2021_08_07_13_38_21_trim](https://user-images.githubusercontent.com/32206511/128589634-e832837a-9d04-48dd-9c0c-e89bd95b3ace.gif)

**Smartphone (please complete the following information):**
 - Device: Redmi S2
 - Android Version : A11
 - Kiwi Browser Version: 93.0.4577.22

**Comment by DUOLabs333** [comment]
This is true even if toolbar is on the top, though that is because there is nowhere for it to scroll to.

**Comment by DUOLabs333** [comment]
This could be fixed by making the keyboard's top border the limit for the "window" in which the suggestions scroll through.
